posthog/tasks/test/test_usage_report.py

```diff
@@ -1074,6 +1075,7 @@ def test_usage_report_hogql_queries(self) -> None:
             assert report.event_explorer_api_rows_read == 0
 
     @also_test_with_materialized_columns(event_properties=["$lib"], verify_no_jsonextract=False)
+    @pytest.mark.skip(reason="Skipping due to flakiness")
```

Comment:
BTW, CI is still failing, probably because of the `@also_test_with_materialized_columns`, Im updating this